### PR TITLE
[Tabs] Update Tabs Theme Section README.md

### PR DESCRIPTION
**Pull request by mohammadcazig**

### CODE REVIEW

File: components/Tabs/README.md [hunk 1]
```diff
@@ -116,6 +116,75 @@ navigation bar. The bar will automatically update with the appropriate styling.
...
+
+#### Objective-C
+
+``` objc
+#import "MaterialTabs+ColorThemer.h"
+#import "MaterialTabs+TypographyThemer.h"
+```
+<!--</div>-->
+
+MDCTabBarColorThemer allows you to theme an app bar with your app's color scheme.
```
Comment (romoore) [inline components/Tabs/README.md]: copy-paste bug? s/app bar/tab bar/

same below

File: components/Tabs/README.md [hunk 2]
```diff
@@ -116,6 +116,75 @@ navigation bar. The bar will automatically update with the appropriate styling.
...
+
+MDCTabBarColorThemer allows you to theme an app bar with your app's color scheme. 
+
+MDCTabBarTypographyThemer allows you to theme an app bar with your app's typography scheme. 
+
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
+``` swift
+let colorScheme: MDCSemanticColorScheme()
+let typographyScheme: MDCTypographyScheme()
```
Comment (romoore) [inline components/Tabs/README.md]: Swift syntax?

File: components/Tabs/README.md [hunk 3]
```diff
@@ -116,6 +116,75 @@ navigation bar. The bar will automatically update with the appropriate styling.
...
+
+let tabBar = MDCTabBar()
+
+// Apply regular style
+MDCTabBarColorThemer.apply(colorScheme, toTabs: tabBar)
+
+// Or Surface Variant
+MDCTabBarColorThemer.apply(colorScheme, toTabs: tabBar)
```
Comment (romoore) [inline components/Tabs/README.md]: Should the variant have a different method signature?

**Review by romoore — APPROVED** [review]

File: components/Tabs/README.md [hunk 4]
```diff
@@ -116,6 +116,76 @@ navigation bar. The bar will automatically update with the appropriate styling.
...
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
+``` swift
+let colorScheme = MDCSemanticColorScheme()
+let typographyScheme = MDCTypographyScheme()
+
+let tabBar = MDCTabBar()
+
+// Apply regular style
+MDCTabBarColorThemer.apply(colorScheme, toTabs: tabBar)
```
Comment (jverkoey) [inline components/Tabs/README.md:160]: I think this has to be applySemanticColorScheme

Comment (mohammadcazig) [reply components/Tabs/README.md:160]: fixed straight to dev since this was already pushed.

**Pull request merged**